examples/training/classification/imagenet/basic_training.py

```diff
@@ -246,12 +246,11 @@ def augment(img, label):
 """
 
 with strategy.scope():
-    model = models.__dict__[FLAGS.model_name]
-    model = model(
-        include_rescaling=True,
-        include_top=True,
+    backbone = models.__dict__[FLAGS.model_name]
+    model = models.ImageClassifier(
```

Comment:
nit: maybe we should just explicitly construct using `from_preset`?

Reply:
I like this idea. I can switch to that paradigm for the next time I run this